spec/ravelin/authentication_mechanisms/magic_link_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe Ravelin::AuthenticationMechanisms::MagicLink do
```

Comment:
Also, please add some test cases for the absence of `phoneNumber` for `transport == "sms"`, and the absence of `email` for `transport == "email"`

Reply:
Added, thanks